src/app/pages/registration/components/register-panel/register-panel.component.html

```diff
@@ -0,0 +1,182 @@
+        [type]="'password'"
+        [placeholder]="buildTranslationKeyForPlaceholders('password') | translate"
+        [isValid]="passwordControl.valid"
+        [isDirty]="passwordControl.dirty && passwordControl.touched">
+      </app-text-input>
+
+      <app-message
+        *ngIf="passwordControl.touched && passwordControl.hasError('minlength')"
+        [type]="'error'"
+        [message]="(buildTranslationKeyForMessages('minLength') | translate) + passwordControl.getError('minlength')['requiredLength']">
+      </app-message>
+
+      <app-message
+        *ngIf="passwordControl.dirty && passwordControl.hasError('required')"
+        [type]="'error'"
+        [message]="buildTranslationKeyForMessages('required') | translate">
+      </app-message>
+    </div>
+
+    <div class="flex flex-col gap-1">
+      <app-text-input
+        [formControlName]="'confirmPassword'"
+        [label]="buildTranslationKeyForForm('confirmPassword') | translate"
+        [type]="'password'"
+        [placeholder]="buildTranslationKeyForPlaceholders('confirmPassword') | translate"
+        [isValid]="confirmPasswordControl.valid"
+        [isDirty]="confirmPasswordControl.dirty && confirmPasswordControl.touched">
+      </app-text-input>
+
+      <app-message
+        *ngIf="confirmPasswordControl.touched && confirmPasswordControl.hasError('minlength')"
+        [type]="'error'"
+        [message]="(buildTranslationKeyForMessages('minLength') | translate) + confirmPasswordControl.getError('minlength')['requiredLength']">
+      </app-message>
+
+      <app-message
+        *ngIf="confirmPasswordControl.dirty && confirmPasswordControl.hasError('required')"
+        [type]="'error'"
+        [message]="buildTranslationKeyForMessages('required') | translate">
+      </app-message>
+
+      <app-message
+        *ngIf="confirmPasswordControl.dirty && confirmPasswordControl.hasError('passwordShouldMatch')"
+        [type]="'error'"
+        [message]="buildTranslationKeyForMessages('passwordMatch') | translate">
+      </app-message>
+    </div>
+  </div>
+
+  <div class="flex items-center justify-end p-2 gap-3">
+    <span class="after:content-['*'] after:ml-0.5 after:text-red-500 block text-sm font-medium text-slate-700">
+      {{ buildTranslationKeyForForm('termsAndConditions') | translate }}
+    </span>
+
+    <app-checkbox
+      [formControlName]="'termsAndConditions'"
+      [isCheckBoxHidden]="false"></app-checkbox>
+  </div>
+
+  <div>
+    <app-button
+      [text]="buildTranslationKeyForForm('signUp') | translate"
+      [disabled]="registerForm.invalid || registerForm.pending"
+      [type]="'primary'">
+    </app-button>
+  </div>
+
+  <div>
+    <span>
+      {{ buildTranslationKeyForForm('alreadyRegistered') | translate }}
+      <strong class="text-blue-600 cursor-pointer" (click)="navigateToLogin()">
+        {{ buildTranslationKeyForForm('loginHere') | translate }}
+      </strong>
```

Comment:
Add the `role="button"` to the elements that are not clickable, but have (click) event to fix this Linter error:
https://excellentwebcheck.com/blogs/how-to-fix-jsx-a11y-click-events-have-key-events